README.md

```diff
@@ -3,31 +3,31 @@
 Rails app for managing the transition of websites to GOV.UK. Specifically, it's for the production of and handling
 of mappings for use with [Bouncer](https://github.com/alphagov/bouncer).
 
-## Dependencies
+## Technical documentation
 
-* Redis
-* PostgreSQL 9.3+ (the app uses materialized views, which were introduced in 9.3).
-  This is included in the Trusty dev VM, which is now the default.
+This is a Ruby on Rails app, and should follow [our Rails app conventions](https://docs.publishing.service.gov.uk/manual/conventions-for-rails-applications.html).
 
-## Running the app
+You can use the [GOV.UK Docker environment](https://github.com/alphagov/govuk-docker) to run the application and its tests with all the necessary dependencies. Follow [the usage instructions](https://github.com/alphagov/govuk-docker#usage) to get started.
 
-The web application itself is run like any other Rails app, for example:
+**Use GOV.UK Docker to run any commands that follow.**
 
-```sh
-script/rails s
-```
-
-In development, you can run sidekiq to process background jobs:
+### Running the tests
 
-```sh
-bundle exec sidekiq -C config/sidekiq.yml
```

Comment:
Sounds like we ought to have a sidekiq worker in the [transition Docker config](https://github.com/alphagov/govuk-docker/blob/master/projects/transition/docker-compose.yml), like we did in https://github.com/alphagov/govuk-docker/pull/461. Out of scope for this PR, obviously, but I wonder if it's worth keeping this in the README as a 'gotcha' until that change to govuk-docker has been made?

Reply:
Good point. The development scenarios for actually running [the worker](https://github.com/alphagov/transition/blob/63e0386498d5960a5ec26fbf1b550c533cab9fc8/app/controllers/import_batches_controller.rb#L34) seem pretty niche to me, so I'm not sure it's even worth adding it to GOV.UK Docker. I've amended to commit/message to preserve the command 👍.